apiserver/facades/agent/secretsmanager/secrets.go

```diff
@@ -219,7 +220,8 @@ func (s *SecretsManagerAPI) RemoveSecrets(args params.DeleteSecretArgs) (params.
 	if err != nil {
 		return params.ErrorResults{}, errors.Trace(err)
 	}
-	if err := provider.CleanupSecrets(model, removedURIs); err != nil {
+	// TODO: include unitTag in params.DeleteSecretArgs for operator uniters?!!!
```

Comment:
Currently, 3.0 is broken the operator style applications due to https://bugs.launchpad.net/juju/+bug/1991213
This PR won't cover the legacy operator-style applications.
We need to revisit once lp:1991213 is fixed.